topology/m4/mixercontrol.m4

```diff
@@ -28,6 +28,47 @@ define(`CONTROLMIXER_OPS',
 `		put STR($4)'
 `	}')
 
+dnl KCONTROL_2CHANNELS(reg)
+define(`KCONTROL_2CHANNELS',
+`KCONTROL_CHANNEL(FL, $1, 0),'
```

Comment:
@RanderWang we dont really use the reg and shift values in our IO handler at all. Should we just remove them so that people dont get confused about why it is like this?

Reply:
it is used in soc-topology.c

Reply:
@RanderWang the question is if it's used in the SOF code.

Reply:
@RanderWang agreed it may be used by other kcontrol handlers. But have you verified the reg and shift values are correct? I still think we should remove them.

Reply:
@ranj063 @plbossart   It is not used in SOF now, but not never. I verify the the reg and shift, no problem. It is used in Asoc for two usages. The first usage is to check the pcm is mono audio or stereo audio. If it is removed, all pcm are mono audio. The second usage is to set hardware register. But for SOF, it is no use.